programs/format/Format.cpp

```diff
@@ -44,6 +44,7 @@ int mainEntryClickHouseFormat(int argc, char ** argv)
 
     boost::program_options::options_description desc = createOptionsDescription("Allowed options", getTerminalWidth());
     desc.add_options()
+        ("query", boost::program_options::value<std::string>(), "query to format")
```

Comment:
```suggestion
        ("query", po::value<std::string>, "query to format")
```

Reply:
Hi! Thanks for your reviewing!  
Is this place should be `po::value<std::string>()` rather than `po::value<std::string>` ?